lib/iris/cube.py

```diff
@@ -3161,15 +3161,35 @@ def __hash__(self):
 
     def __add__(self, other):
         return iris.analysis.maths.add(self, other, ignore=True)
+
+    def __iadd__(self, other):
+        return iris.analysis.maths.add(self, other, ignore=True, in_place=True)
+
     __radd__ = __add__
 
     def __sub__(self, other):
         return iris.analysis.maths.subtract(self, other, ignore=True)
 
+    def __isub__(self, other):
+        return iris.analysis.maths.subtract(self, other,
+                                            ignore=True, in_place=True)
+
     __mul__ = iris.analysis.maths.multiply
     __rmul__ = iris.analysis.maths.multiply
+
+    def __imul__(self, other):
+        return iris.analysis.maths.multiply(self, other, in_place=True)
+
     __div__ = iris.analysis.maths.divide
```

Comment:
Don't panic! We do want this behaviour as all division overrides here call `maths.divide`, which only uses the `truediv` / `itruediv` operator. So actually this should be kept so that it funnels all incoming division operations on cubes into running `maths.divide`, and all is hunky-dory.